BasiliskII/src/SDL/video_sdl.cpp

```diff
@@ -1825,7 +1832,7 @@ static void handle_events(void)
 			// Application activate/deactivate
 			case SDL_ACTIVEEVENT:
 				// Force a complete window refresh when activating, to avoid redraw artifacts otherwise.
-				if (event.active.gain && (event.active.state & SDL_APPACTIVE))
```

Comment:
Your diff looks wrong. For example, this change is already in upstream - as well as a bunch of other changes to this file.

Also, I still see all the slirp changes in the diff?

Reply:
I have exclude video_sdl.cpp.

And, I want to change PR branch, but should I recreate the PR to change the branch?

Reply:
It might be easier to recreate the branch and submit another PR if you're having trouble getting unrelated changes to not show up in the diff.

When reviewing the PR, I'd like to see only the changes you're submitting to be in the diff. Otherwise, it's very confusing.